frontend/src/concepts/pipelines/content/createRun/contentSections/ParamsSection/ParamsSection.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import { Alert, FormGroup, FormSection, TextInput } from '@patternfly/react-core';
+import {
+  CreateRunPageSections,
+  runPageSectionTitles,
+} from '~/concepts/pipelines/content/createRun/const';
+import {
+  InputDefParamType,
+  PipelineVersionKFv2,
+  RuntimeConfigParameters,
+} from '~/concepts/pipelines/kfTypes';
+import { getInputDefinitionParams } from '~/concepts/pipelines/content/createRun/utils';
+import { IntegerInputParam } from './IntegerInputParam';
+import { RadioInputParam } from './RadioInputParam';
+import { JsonInputParam } from './JsonInputParam';
+
+type ParamsSectionProps = {
+  runParams: RuntimeConfigParameters | undefined;
+  version: PipelineVersionKFv2 | null;
+  onChange: (params: RuntimeConfigParameters) => void;
+};
+
+export const ParamsSection: React.FC<ParamsSectionProps> = ({
+  runParams = {},
+  version,
+  onChange,
+}) => {
+  const inputDefinitionParams = getInputDefinitionParams(version);
```

Comment:
this could be moved under the version check becuase it is not used until line 47